test/build_function_arrayofarray_named_tuples.jl

```diff
@@ -0,0 +1,197 @@
+using Symbolics, Test, SparseArrays
```

Comment:
this file isn't in runtests.jl so it's not tested.

Reply:
Sorry, I forgot about that. Just did it.